processor/src/operations/crypto_ops.rs

```diff
@@ -51,10 +51,10 @@ impl Process {
     /// To perform the operation we do the following:
     /// 1. Look up the Merkle path in the advice provider for the specified tree root.
     /// 2. Use the hasher to compute the root of the Merkle path for the specified node.
-    /// 3. Replace the node value with the computed root.
+    /// 3. Copy the stack state over to the next clock cycle with no changes.
```

Comment:
I would maybe put as 3rd bullet point something like: "Verifies that the computed root is equal to the root provided via the stack." and then put the current 3rd bullet point after this.

Reply:
Done